lib/src/rules/sort_constructors_first.dart

```diff
@@ -71,4 +71,14 @@ class _Visitor extends SimpleAstVisitor<void> {
       }
     }
   }
+
+  @override
+  void visitClassDeclaration(ClassDeclaration node) {
+    check(node.members);
+  }
+
+  @override
+  void visitEnumDeclaration(EnumDeclaration node) {
+    check(node.members);
```

Comment:
I'm guessing that this means that enum constants aren't included in `members`.

Reply:
Yep.